test/selenium_tests/framework.py

```diff
@@ -110,7 +110,11 @@ def write_file(name, content, raw=False):
                 buf.write(content.encode("utf-8") if not raw else content)
 
         os.makedirs(target_directory)
-        self.driver.save_screenshot(os.path.join(target_directory, "last.png"))
+        try:
+            self.driver.save_screenshot(os.path.join(target_directory, "last.png"))
+        except Exception:
+            print("Failed to dump screenshot for error debugging.")
+            write_file("screenshot_exception.txt", traceback.format_exc())
         write_file("page_source.txt", self.driver.page_source)
```

Comment:
It seems that in https://jenkins.galaxyproject.org/job/docker-selenium/1685/testReport/selenium_tests.test_library_to_collections/LibraryToCollectionsTestCase/test_list_creation/ , this line now raises an Exception, so it should also go in the try/except block